src/Pagination.js

```diff
@@ -1,9 +1,69 @@
-import React from 'react'
+import Pagination from "react-bootstrap/Pagination";
 
-export default function Pagination() {
-  return (
-    <div>
+export default function PaginationFunction({currentPage, setcurrentPage, totalPages }) {
+  function handlePageChange(page) {
+    setcurrentPage(page);
+  }
+
+  function PaginationFunction ({currentPage, totalPages, handlePageChange}) { //this function returns back the render elements for both buttons and page numbers
```

Comment:
put comments line above the function, the line it often limited to 150letters so we don't have to scroll horizontally

Reply:
Understand moved the comments above function